README.md

```diff
@@ -36,11 +36,29 @@ How To (Plugin Developers)
 <dependency>
     <groupId>com.destroystokyo.paper</groupId>
     <artifactId>paper-api</artifactId>
-    <version>1.15.2-R0.1-SNAPSHOT</version>
+    <version>1.16.1-R0.1-SNAPSHOT</version>
     <scope>provided</scope>
  </dependency>
  ```
 
+##### Or alternatively, with Gradle:
+
+ * Repository:
+```groovy
+repositories {
+    maven {
+        url('https://papermc.io/repo/repository/maven-public/')
+    }
+}
+```
+
+ * Artifact:
+```groovy
+dependencies {
+    compileOnly('com.destroystokyo.paper:paper-api:1.16.1-R0.1-SNAPSHOT')
```

Comment:
```suggestion
    compileOnly 'com.destroystokyo.paper:paper-api:1.16.1-R0.1-SNAPSHOT'
```

Reply:
That actually wasn't my idea, it was suggested by @Proximyst